packages/apps/docs/src/components/Layout/Blog/Blog.tsx

```diff
@@ -90,16 +89,17 @@ export const Blog: FC<IPageProps> = ({
                 )}
               </ArticleMetadataItem>
               <ArticleMetadataItem>
-                {tags &&
-                  tags.map((tag) => (
-                    <Link
-                      className={tagLinkClass}
-                      key={tag}
-                      href={`/tags/${tag}`}
-                    >
-                      <Tag>{tag}</Tag>
-                    </Link>
-                  ))}
+                {tags && (
+                  <TagGroup tagAsChild aria-label={`${title} Tags`}>
```

Comment:
I think the taggroup needs an display:inline-block option for this issue (or something).

![image](https://github.com/kadena-community/kadena.js/assets/4015521/115d27de-13b9-403e-b989-c43759cd561b)

Reply:
fixed 👍